src/prx/converters.py

```diff
@@ -50,7 +50,7 @@ def compact_rinex_obs_file_to_rinex_obs_file(file: Path):
     )
     assert len(crx2rnx_binaries) > 0, "Could not find any CRX2RNX binary"
     for crx2rnx_binary in crx2rnx_binaries:
-        command = f" {crx2rnx_binary} {file}"
+        command = f" {crx2rnx_binary} -f {file}"
```

Comment:
overwrites expanded file if for whatever reason it already exists, otherwise `crx2rnx` will wait (indefinitely) for user input.